app/models/container_quota_item.rb

```diff
@@ -1,3 +1,29 @@
 class ContainerQuotaItem < ApplicationRecord
   belongs_to :container_quota
+  BINARY_SUFFICES = ["Ki", "Mi", "Gi", "Ti", "Pi", "Ei"].freeze
+  SI_SUFFICES = ["K", "M", "G", "T", "P", "E"].freeze
+
+  def normalize_field_for_chargeback(field_sym)
```

Comment:
can you write down as comment for this method example of usage?
what is input and what is output ?